src/popup/components/mnemonicPhrase/ConfirmMnemonicPhrase.tsx

```diff
@@ -36,82 +37,97 @@ const ClearButton = styled(Button)`
   width: 24px;
 `;
 
-const ConfirmMnemonicPhrase = ({
-  mnemonicPhrase,
-}: {
-  mnemonicPhrase: string;
-}) => {
+const convertToWord = (wordKey: string) => wordKey.replace(/-.*/, "");
+
+const ConfirmMnemonicPhrase = ({ words = [""] }: { words: string[] }) => {
   const dispatch = useDispatch();
-  const words = shuffle(mnemonicPhrase.split(" "));
```

Comment:
so we need to wait to fetch the mnemonic phrase, then once we have it, shuffle all the words into a random order. but we want to only do it ONCE and not re-do on re-render. I was doing this w ref's, but it was creating a bit of a mess, i just moved this logic to the parent